near/nep141-locker/src/lib.rs

```diff
@@ -716,16 +716,22 @@ impl Contract {
             .to_log_string(),
         );
 
-        ext_token::ext(token)
-            .with_static_gas(LOG_METADATA_GAS)
-            .with_attached_deposit(ONE_YOCTO)
-            .ft_transfer(fin_transfer.fee_recipient, U128(fee), None)
+        if fee > 0 {
+            PromiseOrValue::Promise(
+                ext_token::ext(token)
+                    .with_static_gas(LOG_METADATA_GAS)
```

Comment:
```suggestion
                    .with_static_gas(FT_TRANSFER_CALL_GAS)
```

I believe that we should use `FT_TRANSFER_CALL_GAS` instead of `LOG_METADATA_GAS`

Reply:
Fixed with FT_TRANSFER_GAS